### DISCUSSION TITLE
VRoid SDK for Unreal Engine 1.0.0 Release Announcement

### DISCUSSION THREAD

**Discussion by Haresaku** [pixiv/vroid-sdk-developers#167] [Announcements]
To all our SDK developers

Thank you for all the valuable feedback you've provided since last year's preview release.
We are excited to announce the release of the stable VRoid SDK for Unreal Engine, featuring support for UE5.4 and a completely redesigned authentication screen UI.

▼New features
- Now supports Unreal Engine 5.4, alongside continued support for Unreal Engine 5.3.
- Experience a completely redesigned authentication UI.
The license information has been upgraded to the stable version.
Now also supports VRM1.0 licenses.
Moreover, with this SDK, it is now possible to create an official application without altering the license UI.
- View StaffPicks character thumbnails even before authentication.
- Updated character selection tab post-authentication.
- Automatically navigate to your character tab if you have one.
- Automatically select your ♡ character tab if you have characters marked with ♡.
- Automatically navigate to the recommended character tab if you don't have any of the above.
- Experience a completely redesigned session management UI for multiplayer.
In addition to creating or joining a session, you now have three options: customize session settings during creation, view session search/list, and quick play.
- Fixed an issue where certain kanji characters appeared garbled.
Fixed a bug where some kanji characters were incorrectly displayed in a Chinese writing style.

▼Other bug fixes
- Fixed a bug where model updates failed when using PlayerController inheriting from VRoidPlayerControllerBase during multiplayer sessions.
- Fixed a bug where the use model button remained enabled upon reopening the license screen after agreeing to the terms and conditions.
- Fixed a build error caused by overlapping naming of an existing plugin and enumeration type during migration to UE5.4.
- Fixed an issue where an error occurred when attempting to access a null object during JSON deserialization.
- Fixed a warning caused by the absence of a cached character when generating a preview character after authentication.

We look forward to your feedback.

Thank you for your support.

https://github.com/user-attachments/assets/4d3994bc-56d1-4f7a-8fa1-39bee77bca8d

**Comment by lvsas** [comment]
Godot support next ?
